### ISSUE TITLE
MacOS.System.Plist has looping error that produces duplicate rows

### ISSUE THREAD

**Issue by chris-coalitionir** [Velocidex/velociraptor#1659]
In the body of the `MacOS.System.Plist` artifact, the `row` parameter of the `PlistDetails` foreach loop calls the `plist` plugin on each glob result rather than just using glob results directly.  This leads to many duplicate rows, specifically one duplicate for each `Content` entry in the `plist` call result:

```sql
LET Plists = SELECT FullPath FROM glob(globs=PlistGlob)

LET PlistDetails =
    SELECT * FROM foreach(
        row=plist(file=FullPath), -- <-- this is in error, should just be Plists ...
        query={ SELECT
                FullPath AS FullPath,
                Mtime,
                plist(file=FullPath) AS Content,  -- <-- because the plist plugin gets called here
                if(condition=Upload_File, then=upload(file=FullPath)) AS Upload
            FROM scope()
        }
    )

SELECT * FROM foreach(row=Plists, query=PlistDetails)
```

You can reproduce this by targeting a single file in the glob, say `/Library/Receipts/InstallHistory.plist`.  You should get one row for that one file, but instead you get `length(plist('/Library/Receipts/InstallHistory.plist').Content)` duplicates.  I believe this can be fixed by removing the plugin call in that first `row` parameter, and just using the glob results (`Plists`).

Thanks to @Seeps for finding this!

**Comment by Seeps** [comment]
Sample screenshot attached:
<img width="1685" alt="sample" src="https://user-images.githubusercontent.com/10315852/159352084-2a8fbc70-0b40-4ad6-a4bc-cc9a6de84027.png">

**Comment by scudette** [comment]
Thanks for reporting - fixed in #1673
